x11rb/src/x11_utils.rs

```diff
@@ -114,23 +115,23 @@ macro_rules! atom_manager {
             $vis fn new<C: $crate::protocol::xproto::ConnectionExt>(
                 _conn: &C,
             ) -> ::std::result::Result<$cookie_name<'_, C>, $crate::errors::ConnectionError> {
+                let names = [
+                    $($crate::__atom_manager_atom_value!($field_name$(: $atom_value)?),)*
+                ];
+                let cookies: Result<Vec<_>, _> = names.into_iter().map(|name| _conn.intern_atom(false, name)).collect();
```

Comment:
```
 _conn.intern_atom(
```

Should we import the `x11rb::protocol::xproto::ConnectionExt` trait here, in case the user hasn't already?